flink/v1.15/flink/src/test/java/org/apache/iceberg/flink/FlinkCatalogTestBase.java

```diff
@@ -65,7 +65,7 @@ public void before() {
 
   @After
   public void clean() {
-    sql("DROP CATALOG IF EXISTS %s", catalogName);
+    dropCatalog(catalogName, true);
```

Comment:
This one is not strictly needed for 1.15, as the change which caused the need for this is introduced to Flink 1.16. But it is ok to have it here too, to have less diverged code

Reply:
yeah. it is a little better to have less diverged code